src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -602,12 +616,13 @@ IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclaratio
   IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'i1(2)')
     Variables: Local_1: i1 As System.Int32()
     Initializer: 
-      IInvalidExpression (OperationKind.InvalidExpression, Type: System.Int32(), IsInvalid) (Syntax: 'As New Integer')
-        Children(1):
-            IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32, IsInvalid) (Syntax: 'New Integer')
-              Arguments(0)
-              Initializer: 
-                null
+      ILocalInitializer (OperationKind.LocalInitializer, IsInvalid) (Syntax: 'As New Integer')
+        IInvalidExpression (OperationKind.InvalidExpression, Type: System.Int32(), IsInvalid) (Syntax: 'As New Integer')
```

Comment:
>IInvalidExpression (OperationKind.InvalidExpression, Type: System.Int32(), IsInvalid) (Syntax: 'As New Integer') [](start = 8, length = 112)

It feels like this node should be marked as IsImplicit.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/22576](https://github.com/dotnet/roslyn/issues/22576)

---
In reply to: [143049961](https://github.com/dotnet/roslyn/pull/22546#discussion_r143049961) [](ancestors = 143049961)